test/playbacks/hls_spec.js

```diff
@@ -80,16 +80,23 @@ describe('HLS playback', () => {
     })
   })
 
-  it('should trigger a playback error if source load failed', function(done) {
-    this.timeout(5000)
+  it('should trigger a playback error if source load failed', function() {
```

Comment:
Here we could use the usual phrase but I'd vote to remove `should` :P

Reply:
Yes, I agree. But the tests are with lot of should on it messages. I fixed the test but didn't change its message. I think a refactor to change all these messages is better.

Reply:
I agree with removing `should`. Using the present tense would make the test declarations more descriptive.